2024-10-05 14:26:47,333 Downloading artifacts
[34m[1mwandb[39m[22m: [32m[41mERROR[39m[49m Unable to fetch artifact with name ssombre-west-governors-university/nyc_airbnb/random_forest_export:prod
Traceback (most recent call last):
  File "/root/miniconda3/envs/mlflow-441126a0e3ca3460d5c03e1cf88eecb78a6a0e10/lib/python3.9/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/root/miniconda3/envs/mlflow-441126a0e3ca3460d5c03e1cf88eecb78a6a0e10/lib/python3.9/site-packages/wandb/apis/public.py", line 1040, in artifact
    artifact = wandb.Artifact._from_name(
  File "/root/miniconda3/envs/mlflow-441126a0e3ca3460d5c03e1cf88eecb78a6a0e10/lib/python3.9/site-packages/wandb/sdk/artifacts/artifact.py", line 256, in _from_name
    raise ValueError(
ValueError: Unable to fetch artifact with name ssombre-west-governors-university/nyc_airbnb/random_forest_export:prod
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/root/Project-Build-an-ML-Pipeline-Starter/components/test_regression_model/run.py", line 73, in <module>
    go(args)
  File "/root/Project-Build-an-ML-Pipeline-Starter/components/test_regression_model/run.py", line 27, in go
    model_local_path = run.use_artifact(args.mlflow_model).download()
  File "/root/miniconda3/envs/mlflow-441126a0e3ca3460d5c03e1cf88eecb78a6a0e10/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 371, in wrapper_fn
    return func(self, *args, **kwargs)
  File "/root/miniconda3/envs/mlflow-441126a0e3ca3460d5c03e1cf88eecb78a6a0e10/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 361, in wrapper
    return func(self, *args, **kwargs)
  File "/root/miniconda3/envs/mlflow-441126a0e3ca3460d5c03e1cf88eecb78a6a0e10/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 2769, in use_artifact
    artifact = public_api.artifact(type=type, name=name)
  File "/root/miniconda3/envs/mlflow-441126a0e3ca3460d5c03e1cf88eecb78a6a0e10/lib/python3.9/site-packages/wandb/apis/normalize.py", line 87, in wrapper
    raise CommError(message, err).with_traceback(sys.exc_info()[2])
  File "/root/miniconda3/envs/mlflow-441126a0e3ca3460d5c03e1cf88eecb78a6a0e10/lib/python3.9/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/root/miniconda3/envs/mlflow-441126a0e3ca3460d5c03e1cf88eecb78a6a0e10/lib/python3.9/site-packages/wandb/apis/public.py", line 1040, in artifact
    artifact = wandb.Artifact._from_name(
  File "/root/miniconda3/envs/mlflow-441126a0e3ca3460d5c03e1cf88eecb78a6a0e10/lib/python3.9/site-packages/wandb/sdk/artifacts/artifact.py", line 256, in _from_name
    raise ValueError(
wandb.errors.CommError: Unable to fetch artifact with name ssombre-west-governors-university/nyc_airbnb/random_forest_export:prod